ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -634,6 +638,11 @@ private void processGetBlobResponse(GetRequestInfo getRequestInfo, GetResponse g
           if (getError == ServerErrorCode.No_Error) {
             handleBody(getResponse.getInputStream());
             chunkOperationTracker.onResponse(getRequestInfo.replicaId, true);
+            if (RouterUtils.isRemoteReplica(routerConfig, getRequestInfo.replicaId)) {
+              logger.trace("Cross colo request successful for remote replica in ",
```

Comment:
1. Why choose to log successful instead of unsuccessful cross-colo responses?
2. Would be good to log specific operation. E.g., "Cross colo GetBlobInfo request successful for remote replica..."